packages/expo-router/src/testing-library/mocks.ts

```diff
@@ -41,9 +35,6 @@ jest.mock('expo-linking', () => {
     addEventListener() {
       return { remove() {} } as any;
     },
-    async getInitialURL() {
```

Comment:
This is deadcode. All tests override the linking config with a custom `getInitialURL`, so the default function (which invokes `Linking.getInitialURL()` is no longer called